src/nupic/math/ArrayAlgo.hpp

```diff
@@ -5241,7 +5142,7 @@ namespace nupic {
                      : 
                      );
 
-  #elif defined(NTA_ARCH_64)
+  #else //64bit
```

Comment:
Actually leaving this up to you, see my comment on the PR: https://github.com/numenta/nupic.core/pull/278#issuecomment-74909035